src/app/components/trophy/thumbnail/thumbnail.ts

```diff
@@ -16,10 +16,10 @@ const imgMapping: any = {
 	'platinum-secret': require('./platinum-secret.png'),
 };
 
-export function getTrophyImg(trophy: BaseTrophy, isAchieved = true) {
+export function getTrophyImg(trophy: BaseTrophy) {
 	// Make sure we don't show thumbnails for secret trophies unless they've
 	// been achieved.
-	if (trophy.has_thumbnail && (!trophy.secret || isAchieved || trophy.is_owner)) {
+	if (trophy.has_thumbnail && (!trophy.secret || trophy.is_achieved || trophy.has_perms)) {
```

Comment:
Consider making this a getter on the trophy model for wether the trophy info should show.